src/index.html

```diff
@@ -4,9 +4,313 @@
+            <div class="card__stars-star"></div>
+          </div>
+          <div class="card__rewiev">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="card__stars">
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+          </div>
+          <div class="card__rewiev">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="card__stars">
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+            <div class="card__stars-star"></div>
+          </div>
+          <div class="card__rewiev">Reviews: 5</div>
+        </div>
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__product-code">Product code: 195434</p>
+        <div class="card__rate">
+          <div class="card__stars">
```

Comment:
```suggestion
          <div class="card__stars stars stars--4">
```
The quantity of yellow stars should depend on the modifier class. So keep styles from the previous task where you have styled stars